bundles/mapping/mapmodule/plugin/layers/LayersPluginClass.ol.js

```diff
@@ -321,6 +322,16 @@ export class LayersPlugin extends AbstractMapModulePlugin {
     }
     notifyLayerVisibilityChanged (layer, inScale, geometryMatch) {
         var event = Oskari.eventBuilder('MapLayerVisibilityChangedEvent')(layer, inScale, geometryMatch);
+
+        if (layer.isVisible()) {
+            if (!inScale) {
+                Messaging.notify(Oskari.getMsg('MapModule', 'layerVisibility.notInScale', { name: layer.getName() }));
```

Comment:
This will affect embedded maps which might use runtime vector layers that don’t have names or at least user-friendly names. I don’t have a good suggestion for this now but I’m very hesitant for merging this as is. I think we need some other solution for this or handle some edge cases at least.